### DISCUSSION TITLE
Please enable code scanning in the repository settings

### DISCUSSION THREAD

**Discussion by wyttenbach** [community/community#126642] [Repositories] [answered]
### Select Topic Area

Question

### Body

I'm trying to upload CodeQL results and am getting the error in the subject.
How do I enable this? In the Repo settings all I see is Dependabot-related stuff.

**Comment by pedrolemos4** [comment, marked as answer]
You need to have your repository set as public for the default settings. 
This article can explain everything step by step and in more detail https://docs.github.com/pt/code-security/code-scanning/enabling-code-scanning/configuring-default-setup-for-code-scanning

**Comment by wyttenbach** [comment]
Thanks for that Pedro.

From: Pedro Lemos ***@***.***>
Sent: Thursday, May 30, 2024 6:37 PM
To: community/community ***@***.***>
Cc: Wyttenbach, Dale R. (halfaker And Associates, Llc) ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [community/community] Please enable code scanning in the repository settings (Discussion #126642)


You need to have your repository set as public for the default settings.
This article can explain everything step by step and in more detail https://docs.github.com/pt/code-security/code-scanning/enabling-code-scanning/configuring-default-setup-for-code-scanning

-

**Reply by pedrolemos4** [reply]
No worries, can you please mark my answer as correct to close the issue?
Thanks :)

**Reply by pedrolemos4** [reply]
You marked yours 😅
